waves/src/Bobtimus.tsx

```diff
@@ -135,9 +155,14 @@ export async function postLoanFinalization(txHex: string) {
         body: JSON.stringify({ tx_hex: txHex }),
     });
 
-    if (res.status !== 200) {
+    if (res.status === 400) {
+        const resp = await res.json();
+        debug("received loan validation error: " + resp.title + resp.detail);
```

Comment:
Nit on using `debug`: To avoid unnecessary string allocations, using formatting specifiers like `%s` would be better.

Reply:
thanks, I'll keep it in mind!